src/app.js

```diff
@@ -12,13 +14,17 @@ function App() {
   const modal = useSelector(function (state) {
     return state.modal.showmodal;
   });
+  const adduser = useSelector(function (state) {
+    return state.modal.showAddusersModal;
```

Comment:
please change `showAddusersModal` to `showAddUsersModal`

Reply:
ofcourse